class TaskFail(Exception):
    """Ошибка тасков"""
